dwave_networkx/drawing/chimera_layout.py

```diff
@@ -97,15 +104,15 @@ def chimera_layout(G, scale=1., center=None, dim=2):
         m = max(idx[0] for idx in chimera_indices.values()) + 1
         n = max(idx[1] for idx in chimera_indices.values()) + 1
         t = max(idx[3] for idx in chimera_indices.values()) + 1
-        xy_coords = chimera_node_placer_2d(m, n, t, scale, center, dim)
+        xy_coords = chimera_node_placer_2d(m, n, t, scale, center, dim, plot_kwargs=plot_kwargs)
 
         # compute our coordinates
         pos = {v: xy_coords(i, j, u, k) for v, (i, j, u, k) in chimera_indices.items()}
 
     return pos
 
 
-def chimera_node_placer_2d(m, n, t, scale=1., center=None, dim=2):
+def chimera_node_placer_2d(m, n, t, scale=1., center=None, dim=2, plot_kwargs=None):
```

Comment:
This functionality needs to be added to https://docs.ocean.dwavesys.com/en/stable/docs_dnx/reference/drawing.html#module-dwave_networkx.drawing.chimera_layout (doc/reference/drawing.rst): 

```
.. autosummary::
   :toctree: generated/

   chimera_layout
   chimera_node_placer_2d
   draw_chimera
```